commit 09e30295cc9c1023c4d738b9a566882df0dbff96
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-18T12:47:47-07:00 (3 months ago)
    
    TOOLS-2293 'ls-missing-release-builds' tool cannot find headnode-joyent, mockcloud, sdc-system-tests builds
    Reviewed by: Chris Burroughs <chris.burroughs@joyent.com>
    Approved by: Chris Burroughs <chris.burroughs@joyent.com>

diff --git a/tools/jr-manifest.json b/tools/jr-manifest.json
index a25ec5b..e519695 100644
--- a/tools/jr-manifest.json
+++ b/tools/jr-manifest.json
@@ -646,7 +646,8 @@
             "name": "sdc-headnode",
             "labels": {
                 "release": true,
-                "mg": "headnode-joyent"
+                "mg": "headnode-joyent",
+                "buildisprivate": true
             }
         },
         {
@@ -801,7 +802,8 @@
             "name": "sdc-system-tests",
             "labels": {
                 "release": true,
-                "mg": "sdc-system-tests"
+                "mg": "sdc-system-tests",
+                "buildisprivate": true
             }
         },
         {
@@ -947,6 +949,7 @@
                 "release": true,
                 "image": "mockcloud",
                 "mg": "mockcloud",
+                "buildisprivate": true,
                 "vm": true
             }
         },
